Vega docs: add note about file type (#3341)
quiltdata/quilt: docs/Catalog/VisualizationDashboards.md

```

For both Vega and Vega Lite you may specify relative paths to package files

> as data sources and the Quilt catalog will resolve the same. Vega treats this

Reviewer: ```suggestion
as data sources and the Quilt catalog correctly resolves them. Vega treats any
```

> data source as JSON by default. Please,
> [specify the file type](https://vega.github.io/vega-lite/docs/data.html#format)
> if you want to use anything than JSON. Example: 

Reviewer: ```suggestion
data source as JSON by default. If you wish to use a different format than JSON, please 
[specify the file type](https://vega.github.io/vega-lite/docs/data.html#format).
For example: 
```

```json

> {
>   "data": "./datasource.csv"
>   "format": {
>     "type": "csv"
>   }
> }

Reviewer: ```suggestion
{
  "data": {
    "url": "./datasource.csv",
    "format": {
      "type": "csv"
    }
}
```

Author: oops
